docs/tutorial/images.rst

```diff
@@ -29,14 +29,32 @@ Or just pick a single one::
 Scan data and details
 ---------------------
 
-You can access the raw image data::
+You can access the raw image data directly. For a `Scan` with only a single frame::
 
     rgb = scan.rgb_image  # matrix with `shape == (h, w, 3)`
     blue = scan.blue_image  # single color so `shape == (h, w)`
 
     # Plot manually
     plt.imshow(rgb)
 
+For scans with multiple frames::
+
+    # returned data has `shape == (n_frames, h, w, 3)`
+    rgb = multiframe_scan.rgb_image
+    # returned data has `shape == (n_frames, h, w)`
+    blue = multiframe_scan.blue_image
+
+    # Manually plot the RGB image of the first frame.
+    plt.imshow(rgb[0, :, :, :])
+
+We can also slice out a subset of frames from an image stack::
+
+    sliced_scan = multiframe_scan[5:10]
+
+This will return a new `Scan` containing data equivalent to:
+
+    multiframe_scan.rgb_image[5:10, :, :, :]
```

Comment:
Depending on which PR goes in first, you may want to update these to the new API.